tests/planning/d.spec.lua

```diff
@@ -0,0 +1,21 @@
+-- luacheck: globals describe it SKIP
+
+return function()
+	describe("test4", function()
+		it("test5", function()
+		end)
+
+		it("test6", function()
+		end)
+	end)
+
+	describe("test4", function()
+		-- Duplicate describe blocks should get merged.
+		it("test5", function()
+			-- Duplicate it blocks will get overwritten.
+		end)
```

Comment:
In `tests/failing/duplicateIt.spec.lua` there is a test that asserts that duplicate it blocks are not overwritten. Is that behavior inconsistent with the one here?

Reply:
Hmm... seems so. I'll have to check why they both seem to be passing.

Reply:
I bet it's just that my test here isn't counting the duplicates properly.

Reply:
Yep. Fixed.